pir/cpp/payload.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
```

Comment:
The proto should probably be in a separate directory to cpp. How about pir/proto ?